.github/workflows/coverage.yml

```diff
@@ -0,0 +1,45 @@
+name: coverage
+
+on:
+  push:
+    branches:
+      - master
+      - TODO
+
+jobs:
+  linux:
+    runs-on: ubuntu-16.04
+    steps:
+    - uses: actions/checkout@v2.0.0
+    - name: Cache
+      uses: actions/cache@v1.0.3
+      with:
+        path: build
+        key: ${{ matrix.os }}-coverage
+
+    - name: Install Depends
+      run: sudo apt install zlib1g-dev libssl-dev build-essential lcov
+
+    - name: Build And Test
+      shell: bash
+      run: |
+          export JOBS=20
+          export CTEST_PARALLEL_LEVEL=$JOBS
+          export CTEST_OUTPUT_ON_FAILURE=1
+          mkdir -p build
+          cd build
+          cmake .. -DEVENT__COVERAGE=ON -DCMAKE_BUILD_TYPE=Debug
+          make -j $JOBS
+          make verify_coverage
+
+    - name: Coveralls GitHub Action
+      uses: coverallsapp/github-action@v1.0.1
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Does secrets exported for PRs?

Reply:
`secrets.GITHUB_TOKEN` exists in all repos, People don't need to do any work for it.

Reply:
I meant another secret variable